### ISSUE TITLE
Unable to see/dump Wii VC titles

### ISSUE THREAD

**Issue by AdroitErudite** [emiyl/dumpling#40]
I am unable to see any Wii VC titles in the using either Aroma or Tiramisu environments. They do not appear in the digital titles or the MLC dump. Neither environment asks for compatibility mode after fresh install and all other Wii U titles are found/dump fine.

**Comment by wayWardLam** [comment]
I am also having this same issue as I attempt to dump the Original Super Smash Bros from my vWii VC

**Comment by TheEnhas** [comment]
This doesn't have any access to original Wii VC games or Wii mode, so in hindsight I should've gotten as many "upgrades" as possible to the Wii U versions before the eShop closed.

**Comment by Crementif** [comment]
Sorry that I'm just responding to this issue. Are both of you referring to games you bought through the Wii (not Wii U) eshop? Because that is unfortunately not something I really added support for. The only Wii games Dumpling can really access are ones that are present on the Wii U filesystem.

**Comment by AdroitErudite** [comment]
Yeah, this was for virtual console games in the virtual Wii. Thank you for responding though.

**Comment by Crementif** [comment]
Ah, that sucks. I think adding Wii nand support is kinda out of scope for now.
